Makefile

```diff
@@ -0,0 +1,21 @@
+include .env
+
+# The target directory is used for setting where the output zip files will end up
+# You can override this with an environment variable, ex
+# TARGET_DIR=my_custom_directory make deploy
+# Alternatively, you can set this variable in the .env file
+TARGET_DIR ?= deploy/
+
+.PHONY: install
+install:
+	# Building project
+	mvn clean install package -DskipTests
+	# Packaging Mage.Client to zip
+	cd Mage.Client && mvn assembly:assembly -DskipTests
+	# Packaging Mage.Server to zip
+	cd Mage.Server && mvn assembly:assembly -DskipTests
```

Comment:
Tests aren't used in package/assembly phase, so no needs in skip param here

Reply:
Removed skips